src/sentry/static/sentry/app/views/organizationDashboard/teamAvatars.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import {Flex} from 'grid-emotion';
+
+import Avatar from '../../components/avatar';
+
+const MAX_VISIBLE_AVATARS = 5;
+const AVATAR_SIZE = 28;
+
+export default class TeamAvatars extends React.Component {
+  static propTypes = {
+    members: PropTypes.array.isRequired,
+  };
+
+  render() {
+    const members = this.props.members.slice(0, MAX_VISIBLE_AVATARS);
+    const numCollapsedMembers = this.props.members.length - members.length;
+
+    return (
+      <AvatarList>
+        {members.map(member => {
+          return <Avatar key={member.id} user={member} size={AVATAR_SIZE} />;
+        })}
+        {!!numCollapsedMembers && (
+          <CollapsedMembers>{numCollapsedMembers}</CollapsedMembers>
+        )}
+      </AvatarList>
+    );
+  }
+}
+
+const AvatarList = styled(({...props}) => <Flex direction="row-reverse" {...props} />)`
+  .avatar {
```

Comment:
hmm I wonder if we could use a component selector here? e.g. ${Avatar} instead of `.avatar`

If not, I think it'd be better to pass this using the `css` prop to `<Avatar>` so we don't need to rely on the `avatar` class name.